tests/robot-tests/tests/libs/common.robot

```diff
@@ -477,6 +475,10 @@ user clicks link
     [Arguments]    ${text}    ${parent}=css:body
     user clicks element    link:${text}    ${parent}
 
+user clicks link by visible text
+    [Arguments]    ${text}    ${parent}=css:body
+    user clicks element    xpath:.//a[text()="${text}"]    ${parent}
```

Comment:
I'm surprised that the `xpath` selector is working compared to the `link` selector used by `user clicks link`. Would have thought these are basically identical in what they're doing.

If there is actually a difference here, would suggest scrapping this new keyword and change `user clicks link` to use the xpath selector instead. It's simpler and if it also covers cases that `link` can't, then it'd be more beneficial to be there instead.